packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -123,8 +123,14 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
         testID,
         ...rest,
       },
-      content: { children: text },
-      icon: createIconProps(icon),
+      content: {
+        accessible: false,
```

Comment:
You might want to filter to only set this on macOS so as to not affect win32 and Android

Reply:
I think these changes make sense for win32 and we're already doing it for Menu V1: https://github.com/microsoft/fluentui-react-native/blob/7943ac41f1e71920e3f959c73aab6ebe0f43fdbc/packages/components/Menu/src/MenuItem/MenuItem.tsx#L53C55-L53C55

Reply:
Oh, perfect! I guess that means the bug doesn't repro on MenuV1 then for macOS?

Reply:
Talked offline, seems not.

Reply:
Correct, MenuV1 doesn't give us "interactive" as intended.